### PR TITLE
layouts, tooltip, drawer integration with meshmap

### PR DESCRIPTION
**Pull request by vineethvanga18**
Signed-off-by: vineethvanga18 <reddy.8@iitj.ac.in>

**Description**
Integration of layouts, tooltip and drawer 

**Notes for Reviewers**
Prometheus and grafana dashboards are not yet covered in this PR, will make a PR once I complete them.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by dhruv0000 — COMMENTED** [review]
Looks good to go as a boilerplate. Might tweak it a bit later as we define a proper UI flow.

**Comment by kushthedude** [comment]
On hold until meshmap is in sync with master.

File: ui/components/visualize/MesheryVisualizeComponent.js [hunk 1]
```diff
@@ -128,88 +161,989 @@ class MesheryVisualizeComponent extends React.Component {
...
     //Checkout the docs for JSON format https://js.cytoscape.org/#notation/elements-json
     const elements = {
-
+      
       nodes: [
         {
           data: {
-            id: 'ingress_gateway',
-            label: 'Ingress Gateway'
+            app: "details",
```
[the remark this answers is not in the document]
Comment (vineethvanga18) [reply ui/components/visualize/MesheryVisualizeComponent.js:180]: Oh I forgot to do that. sure @kushthedude.

**Comment by vineethvanga18** [comment]
@leecalcote @kushthedude merged and resolved conflicts

**Review by leecalcote — APPROVED** [review]
Thanks you, Vineet

File: ui/components/visualize/MesheryVisualizeComponent.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/visualize/MesheryVisualizeComponent.js]: @vineethvanga18 was this addressed? I’m on my mobile phone and having a hard time confirming.

[the remark this answers is not in the document]
Comment (vineethvanga18) [reply ui/components/visualize/MesheryVisualizeComponent.js]: Yes @leecalcote

[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/visualize/MesheryVisualizeComponent.js]: Ok. Good deal.

**Pull request merged**